docs/contributor/go_version.md

```diff
@@ -0,0 +1,3 @@
+# Go Version
+
+Claircore endeavors to declare a `go` version in the go.mod directive that is the minimum required for all the language features to work as expected. The `go` version will also be updated if there is deemed to be a security fix that could directly affect claircore's codebase.
```

Comment:
```suggestion
Claircore endeavors to declare a `go` version in the go.mod directive that is the minimum required for all the language features to work as expected.
The `go` version will also be updated if there is deemed to be a security fix that could directly affect claircore's codebase.
```

I've been trying to get the markdown into one sentence (or overly long clause of mine) per line, so the diffs are easier when we inevitably have to update them.

Reply:
updated will just merge when CI goes green